connect/runtime/src/test/java/org/apache/kafka/connect/runtime/rest/resources/ConnectorsResourceTest.java

```diff
@@ -259,6 +295,25 @@ public void testGetConnectorConfig() throws Throwable {
         PowerMock.verifyAll();
     }
 
+    @Test
+    public void testValidateConfig() throws Throwable {
+        herder.validateConfigs(EasyMock.eq(WorkerTestConnector.class.getName()), EasyMock.eq(props));
+
+        PowerMock.expectLastCall().andAnswer(new IAnswer<ConfigInfos>() {
```

Comment:
You and use `andReturn` here instead, it's a lot more concise (and you could technically put it on the previous line by doing `EasyMock.expect(...).andReturn(CONFIG_INFOS)`).